src/client/components/OrdersSummary/OrderComponent.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './OrdersSummary.styles.css';
+import moment from 'moment-timezone';
+
+export default function OrderComponent({ order, details }) {
+  const [fullView, setFullView] = React.useState(false);
+  const viewOrder = () => {
+    setFullView(!fullView);
+  };
+
+  return (
+    <div>
+      <li>
+        <div className="flex-container">
+          <div>
+            <h3>
+              Order No: {order.order_number}
+              <span> ({order.nr_of_items} items)</span>
+            </h3>
+          </div>
+          <div>
+            <p>{moment(order.updated_at).format('DD-MM-YYYY')}</p>
+            <p>Kr. {order.total_amount}</p>
+          </div>
+          <div>
+            {' '}
+            <button type="button" onClick={viewOrder}>
+              {fullView ? <p>&#x25BC;</p> : <p>&#x25B2;</p>}
```

Comment:
I whould swap this arrows they look in the wrong order. But for the rest the storybook is working 👍